test/__mocks__/serverlessMock.js

```diff
@@ -0,0 +1,52 @@
+const Serverless = require('serverless');
+
+
+const customs = [
+  {
+    dynamodbAutoBackups: {
+      backupRate: 'rate(5 minutes)',
+      source: 'src/backups.handler',
+    },
+  },
+  {
+    dynamodbAutoBackups: {
+    },
+  },
+  {
+    dynamodbAutoBackups: {
+      source: 'src/backups.handler',
+      backupRemovalEnabled: true
+    },
+  },
+  {
+    dynamodbAutoBackups: {
+      backupRate: 'rate(5 minutes)',
+      source: 'src/backups.handler',
+      backupRemovalEnabled: true
+    },
+  },
+  {
+    dynamodbAutoBackups: {
+      backupRate: 'rate(5 minutes)',
+      source: 'src/backups.handler',
+      backupRemovalEnabled: true,
+      backupRetentionDays: 15
+    },
+  },
+  {
+    dynamodbAutoBackups: {
```

Comment:
duplicated with previous

Reply:
Okay, I get it how you do your testing.

Don't do it that way, it's overly complicated. You shouldn't use an array, but rather a map with scenarios, then for each test you select a specific scenario.

```js
const customs = {
simpleScenario: {
    dynamodbAutoBackups: {
      backupRate: 'rate(5 minutes)',
      source: 'src/backups.handler',
    },
  },
emptyScenario: {
    dynamodbAutoBackups: {
    },
  }
}
```

This way you can reuse the same scenario without duplicating, and change the tests orders without breaking the whole thing.

Avoid dynamic test in general, it often makes a mess. :)